Assets/Scripts/RagdollController.cs

```diff
@@ -1,5 +1,6 @@
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.Rendering.Universal;
```

Comment:
this is required for DecalProjector?

Reply:
Yes, DecalProjector inherits from UnityEngine.Rendering.Universal